src/charts/createHorizontalBars.js

```diff
@@ -123,9 +123,7 @@ export function createHorizontalBars(this_, d) {
           );
         d3select(chartContainer)
           .append('div')
-          .html(
-            `<em>This group does not contain any of the first 5 most prevalent levels of ${d.value_col}</em>.<br><br>`
-          );
+          .html(`<em>All values missing in this group.</em>.<br><br>`);
```

Comment:
styling is off:

![image](https://user-images.githubusercontent.com/5428548/28882275-caa47ed4-7778-11e7-86c8-abaf9a544833.png)